shuup/front/utils/product.py

```diff
@@ -156,23 +157,27 @@ def get_orderable_variation_children(product, request, variation_variables, supp
     orderable_variation_children = OrderedDict()
     orderable = 0
 
-    for combo_data in product.get_all_available_combinations():
+    product_queryset = product.variation_children.visible(
```

Comment:
won't the `ProductQuerySet._visible` default parameter `invisible_modes=[ProductMode.VARIATION_CHILD]` be a problem here?

Reply:
That is only problem when using "listed" and "searchable" querysets since with those we do not want to list or search though children. As you see here https://github.com/shuup/shuup/blob/master/shuup/core/models/_products.py#L209